utils/awesome-client

```diff
@@ -53,15 +51,14 @@ a_dbus_send()
     fi
 }
 
-FATAL_ERRORS="$ISATTY"
 if [ $# -ne 0 ]
 then
-    FATAL_ERRORS="1"
     for arg in "$@" ; do
         a_dbus_send "$arg"
     done
-elif [ "$ISATTY" = 0 ]
+elif [ -t 1 ]
```

Comment:
Hm. Good point. But Travis is definitely mixing up the cases here and using the wrong one.